api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -161,10 +166,13 @@
 
         Cohort cohort = cohortReviewService.findCohort(cohortId);
         //this validates that the user is in the proper workspace
-        cohortReviewService.validateMatchingWorkspace(workspaceNamespace, workspaceId, cohort.getWorkspaceId(), WorkspaceAccessLevel.WRITER);
+        Workspace workspace = cohortReviewService.validateMatchingWorkspace(workspaceNamespace,
+            workspaceId, cohort.getWorkspaceId(), WorkspaceAccessLevel.WRITER);
 
+        CdrVersionContext.setCdrVersion(workspace.getCdrVersion());
```

Comment:
So when setting the cdr version in the context, does this mean that all dao/repo calls with in this scope will run in that cdr version of that database? What if i make a call to a dao/repo that should pull data from the workbench?

Reply:
It only applies to everything under org.pmiops.workbench.cdr; see basePackages under @EnableJpaRepositories on CdrDbConfig. So you still will be able to use workbench.db repos with the workbench DB normally.